lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -1,8 +1,8 @@
 # Configurable Logo Used for CartoDB export
 APPLICATION_LOGO_URL: 'http://geoblacklight.org/images/geoblacklight-logo.png'
 
-# CartoDB OneClick Service https://cartodb.com/open-in-cartodb/
-CARTODB_ONECLICK_LINK: 'http://oneclick.cartodb.com/'
+# Carto OneClick Service https://carto.com/engine/open-in-carto/
+CARTO_ONECLICK_LINK: 'http://oneclick.cartodb.com/'
```

Comment:
Just tried it. Works with `carto.com` as well. But I'm OK leaving it as is because that's what's in the docs.